src/Compilers/VisualBasic/Portable/CommandLine/VisualBasicCommandLineParser.vb

```diff
@@ -455,6 +457,24 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                             End If
                             Continue For
 
+                        Case "refout"
+                            Dim unquoted = RemoveQuotesAndSlashes(value)
```

Comment:
>Dim unquoted = RemoveQuotesAndSlashes(value) [](start = 28, length = 44)

We are not doing this for ```out``` and other similar switches (that deal with paths). Should we?